src/main/java/com/passbook/sparkeighteen/config/SwaggerConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.passbook.sparkeighteen.config;
+
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+    @Bean
+    public Docket api() {
+        return new Docket(DocumentationType.SWAGGER_2)
+                .apiInfo(getApiInfo())
+                .select()
+                .apis(RequestHandlerSelectors.withClassAnnotation(RestController.class))
+                .paths(PathSelectors.any())
+                .build();
+    }
+
+    private ApiInfo getApiInfo() {
+
+        return new ApiInfoBuilder()
+                .title("Swagger API Doc")
+                .description("More description about the API")
```

Comment:
Can be something else

Reply:
No sir swagger configure krne ke bad ka msg hai bas.
<img width="459" alt="Screenshot 2022-06-13 at 11 57 03 AM" src="https://user-images.githubusercontent.com/98724071/173292763-f41b35f8-2f70-4848-9196-2b9f6a168a45.png">